autode/conformers/conformer.py

```diff
@@ -94,3 +99,69 @@ def optimise(self,
             self.atoms = None
 
         return None
+
+    @property
+    def coordinates(self) -> Optional[Coordinates]:
+        """Coordinates of this conformer"""
+        return self._coordinates
+
+    @coordinates.setter
+    def coordinates(self, value: 'numpy.ndarray'):
+        """Set the coordinates of this conformer"""
+        if self._parent_atoms is None:
+            raise ValueError('Conformer has no parent atoms. Setting the '
+                            'coordinates will leave the atoms undefined')
+
+        self._coordinates = Coordinates(value)
+
+    @property
+    def atoms(self) -> Optional[Atoms]:
+        """
+        Atoms of this conformer are built from the parent atoms and the
+        coordinates that are unique to this conformer.
+        """
+
+        if self._parent_atoms is None or self._coordinates is None:
+            return None
+
+        atoms = Atoms()
+        for parent_atom, coord in zip(self._parent_atoms, self._coordinates):
+            atom = parent_atom.copy()
+            atom.coord = coord
+
+            atoms.append(atom)
+
+        return atoms
+
+    @atoms.setter
+    def atoms(self,
+              value: Optional[Atoms]):
+        """
+        Set the atoms of this conformer.
+
+        If None then set the corresponding coordinates of this conformer to
+        None (such that self.atoms is None). If this conformer has coordinates
+        then set those from the individual atomic coordinates otherwise
+        set the coordinates as a batch
+        """
+
+        if value is None:
+            self._coordinates = None
+            return  # Nothing to do
```

Comment:
Is it ok not to return anything?

Reply:
It is okay, and it will still return `None` and it is style consistent with the rest of the code base.. but it's not as readable as it could be 👌